cfme/tests/test_rest.py

```diff
@@ -263,6 +263,17 @@ def test_server_info_href(appliance):
         assert 'id' in appliance.rest_api.get(appliance.rest_api.server_info[item])
 
 
+@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+def test_default_region(appliance):
+    """Check that the default region is present.
+
+    Metadata:
+        test_flag: rest
+    """
+    reg = appliance.rest_api.collections.regions.get(region=0)
```

Comment:
@mfalesni Are you saying that region 0 might not be present at all in manageiq? On my appliances there's just one region by default and it's always 0 (it's value of the "region" attribute, not id of the resource).